frappe/www/login.html

```diff
@@ -70,20 +70,29 @@ <h6>{{ _("Or login with") }}</h6>
 </section>
 <section class='for-signup'>
 	<div class="login-content page-card" style="margin-top: 20px;">
-	<form class="form-signin form-signup hide" role="form">
-		<div class="page-card-head">
-			<span class="indicator blue" data-text="{{ _('Sign Up') }}"></span>
+	{%- if not disable_signup -%}
+		<form class="form-signin form-signup hide" role="form">
+			<div class="page-card-head">
+				<span class="indicator blue" data-text="{{ _('Sign Up') }}"></span>
+			</div>
+			<input type="text" id="signup_fullname"
+				class="form-control" placeholder="{{ _('Full Name') }}" required autofocus>
+			<input type="email" id="signup_email"
+				class="form-control" placeholder="{{ _('Email Address') }}" required>
+			<button class="btn btn-sm btn-primary btn-block btn-signup" type="submit">{{ _("Sign up") }}</button>
+		</form>
+	{%- else -%}
+		<div class='page-card-head'>
+			<span class='indicator darkgrey'>{{_("Signup Disabled")}}</span>
 		</div>
-		<input type="text" id="signup_fullname"
-			class="form-control" placeholder="{{ _('Full Name') }}" required autofocus>
-		<input type="email" id="signup_email"
-			class="form-control" placeholder="{{ _('Email Address') }}" required>
-		<button class="btn btn-sm btn-primary btn-block btn-signup" type="submit">{{ _("Sign up") }}</button>
-	</form>
+		<p>{{_("Signups have been disabled for this website.")}}</p>
```

Comment:
```suggestion
		<p>{{_("Sign ups have been disabled for this website.")}}</p>
```

Reply:
Signup is an actual word, which in this case is a noun